lib/td/command/job.rb

```diff
@@ -74,6 +74,7 @@ def job_list(op)
 
     rows = []
     jobs.each {|job|
+      job.auto_update_status = false
```

Comment:
Don't we need to make this configurable by command line option like `td job:list --update-status=false`?

Reply:
This code is executed just 0.xxx seconds after the previous index call. The difference won't update major statuses but takes many extra HTTP calls. I don't think it's worth customizable.

Reply:
I see 👍